### ISSUE TITLE
django migrate problem when I  delete some content types

### ISSUE THREAD

**Issue by lifeignite** [Miserlou/Zappa#1136]
if some column or data can be changed or deleted while migrating,  django warns like below.

> Any objects related to these content types by a foreign key will also
> be deleted. Are you sure you want to delete these content types?
> If you're unsure, answer 'no'.
> 
> Type 'yes' to continue, or 'no' to cancel:

but, as I know, I cannot input yes or no using zappa. django provide `--noinput` options, but it wasn`t work because this option selects 'no' automatically. 
 I want to migrate using `zappa manage migrate`. what should I do?

**Comment by Miserlou** [comment]
Excellent issue. We might need to add the ability to pass additional arguments to `zappa manage`, or, you could write a custom migration command and add it to https://github.com/Miserlou/zappa-django-utils/

Want to work on either of those? The first might make the most sense but I'm also trying to improve the utility of the subproject.

**Comment by VitorFalcao** [comment]
Hi, I would like to start contributing to this project! Is this feature implementation recommended for new contributors?

**Comment by bendog** [comment]
My suggestion would be to give it a go and see if you can make sense of it! 
I'm pretty sure the help wanted tag is a definite request for help :)

**Comment by Miserlou** [comment]
Yep, any of the `hacktoberfest` and `help wanted` tags should be easy(ish)-to-chew issues for newcomers! Please make sure you also read the CONTRIBUTING and PULL REQUEST guides/issue templates before submitting!

**Comment by VitorFalcao** [comment]
Great! I am on it then! But what should be done: "add the ability to pass additional arguments to zappa manage, or, you could write a custom migration command and add it to https://github.com/Miserlou/zappa-django-utils/" ? Is the former or latter preferable?

**Comment by colecrtr** [comment]
I believe this should be closed. I'm not sure if the following was implemented after this issue was created but I was able to successfully execute the `collectstatic` command with `--no-input` by doing:
    
    zappa manage <stage> "collectstatic --no-input"
